cirq/contrib/quimb/mps_simulator_test.py

```diff
@@ -248,9 +278,15 @@ def test_empty_step_result():
 
 def test_state_equal():
     q0, q1 = cirq.LineQubit.range(2)
-    state0 = ccq.mps_simulator.MPSState(qubit_map={q0: 0}, rsum2_cutoff=1e-3, sum_prob_atol=1e-3)
-    state1a = ccq.mps_simulator.MPSState(qubit_map={q1: 0}, rsum2_cutoff=1e-3, sum_prob_atol=1e-3)
-    state1b = ccq.mps_simulator.MPSState(qubit_map={q1: 0}, rsum2_cutoff=1729.0, sum_prob_atol=1e-3)
+    state0 = ccq.mps_simulator.MPSState(
+        qubit_map={q0: 0}, rsum2_cutoff=1e-3, sum_prob_atol=1e-3, grouping=None
```

Comment:
Is explicit assignment of `grouping` here a style choice to clarify test behavior? If not, it's fine to remove them and rely on the default.

Same elsewhere in this file.

Reply:
Removed here and elsewhere.